sqlalchemy_risingwave/base.py

```diff
@@ -20,7 +20,7 @@
     "varchar": sqltypes.VARCHAR,  # DataType::Varchar
     "time without time zone": sqltypes.Time,  # DataType::Time
     "timestamp without time zone": sqltypes.TIMESTAMP,  # DataType::Timestamp
-    "timestamp with time zone": sqltypes.TIMESTAMP(True),  # DataType::Timestampz
```

Comment:
Why change this arguments? We do not need True or False for the timestamp type?

Reply:
It's moved to line 93. So we can store only type, no need to store object in `_type_map`.